content/blog/2023-09-27-epidatr.Rmd

```diff
@@ -0,0 +1,96 @@
+ 
+output:
+  blogdown::html_page:
+    toc: true
+---
+
+The [Delphi Epidata API](https://cmu-delphi.github.io/delphi-epidata/) provides real-time access to epidemiological surveillance data for influenza, COVID-19, and other diseases from both official government sources such as the [Center for Disease Control (CDC)](https://www.cdc.gov/datastatistics/index.html), private partners such as [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Change Healthcare](https://www.changehealthcare.com/), and other public datasets like [Google Trends](https://console.cloud.google.com/marketplace/product/bigquery-public-datasets/covid19-search-trends). It is built and maintained by the Carnegie Mellon University [Delphi research group](https://delphi.cmu.edu/).
+
+Today we introduce the R package "`epidatr`", available [on CRAN](https://cloud.r-project.org/web/packages/epidatr/index.html), with the source and development [on github](https://github.com/cmu-delphi/epidatr).
+
+This package is designed to streamline the downloading and usage of data from the Delphi Epidata API. It provides a simple R interface to the API, including functions for downloading data, parsing the results, and converting the data into a tidy format. The API stores a historical record of all data, including corrections and updates, which is particularly useful for accurately backtesting forecasting models. We also provide packages for downstream data processing ([epiprocess](https://github.com/cmu-delphi/epiprocess)) and modeling ([epipredict](https://github.com/cmu-delphi/epipredict)).
+
+## Usage
+
+```R
+library(epidatr)
+# Obtain the smoothed covid-like illness (CLI) signal from the Facebook survey as it was on April 10, 2021 for the US
+epidata <- covidcast(
+  source = "fb-survey",
+  signals = "smoothed_cli",
+  geo_type = "nation",
+  time_type = "day",
+  geo_values = "us",
+  time_values = epirange(20210101, 20210601),
+  as_of = "2021-06-01")
+epidata
+```
+
+```
+# A tibble: 6 × 15
+  geo_value signal       source   geo_type time_type time_value
+  <chr>     <chr>        <chr>    <fct>    <fct>     <date>
+1 us        smoothed_cli fb-surv… nation   day       2021-04-05
+2 us        smoothed_cli fb-surv… nation   day       2021-04-06
+3 us        smoothed_cli fb-surv… nation   day       2021-04-07
+4 us        smoothed_cli fb-surv… nation   day       2021-04-08
+5 us        smoothed_cli fb-surv… nation   day       2021-04-09
+6 us        smoothed_cli fb-surv… nation   day       2021-04-10
+# ℹ 9 more variables: direction <dbl>, issue <date>,
+#   lag <int>, missing_value <int>, missing_stderr <int>,
+#   missing_sample_size <int>, value <dbl>, stderr <dbl>,
+#   sample_size <dbl>
+```
+
+## Installation
+
+You can install the stable version of this package from CRAN:
+
+``` r
```

Comment:
Same problem as described above, i think this should be un-collapsed by default.